api/v1alpha1/groupversion_info.go

```diff
@@ -19,17 +19,31 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"reflect"
```

Comment:
This piece of code is generated by kubebuilder. I suggest to keep it because we may generate more crds on it.